dom0/sd-svs-disp-files.sls

```diff
@@ -19,3 +19,18 @@ sd-svs-disp-install-mimetype-handler-package:
       - securedrop-workstation-svs-disp
     - require:
       - sls: fpf-apt-test-repo
+
+paxctld:
+  service.running:
+    - enable: True
+    - reload: True
+
+sd-svs-disp-install-libreoffice:
+  pkg.installed:
+    - name: libreoffice
+    - retry:
+        attempts: 3
+        interval: 60
+    - install_recommends: False
+    - require:
+      - service: paxctld
```

Comment:
Given requested changes to task name, this requirement name should be updated. Better yet, if we ensure paxctld is running in a parent state, then we needn't redeclare it here.